src/mono/mono/mini/monovm.h

```diff
@@ -17,4 +17,9 @@ monovm_execute_assembly (int argc, const char **argv, const char *managedAssembl
 MONO_API int
 monovm_shutdown (int *latchedExitCode);
 
+int
```

Comment:
Does this need to be marked with `MONO_API`?

Reply:
It could be, but there's no need.  Marking it as a `MONO_API` would allow the mobile host implementations to call it directly - I'm not sure if that's necessary at this stage since this is primarily aimed at enabling hostfxr and hostpolicy.